examples/diode_chambers/Readme.md

```diff
@@ -0,0 +1,47 @@
+Evaluation of the example
+===========================
+Overall it works well and it is relatively fast given 200 neurons
+with wall-interaction.
+I have tested only diode_chambers.py not the generate_grraph.py
+file.
+
+I tested with these gc_models.
+Graph results are saved into results folder
+
+gf-po-nm
+--------
+The generated neurons go across the diode in both directions,
+since the amount of crossing axons is pretty low the resulting
+network is symmetrical.
+
+gf-srf-nm
+---------
+With this model it works much better, the neurons from
+left->right are much more than the reverse.
+It takes much longer, about 10 minutes.
+
+gf-srf-nwa
+----------
+With this models way more neurons get in the tubes (I think it is because the average between the tubes walls make the gc to go straight)
+I pushed the picture on Imgurrr: http://i.imgur.com/hgCjxfe.png
+I haven't save the plots for this run.
+
+
+
+
+Bugs:
+====
+1. Using same kernel seeds causes different behavior: the position are generated randomly and setting Python random.seed does not solve the problem.
+
+2. The simulation sometimes (not reproducible because it depends on the random generatorr) stuck in this error. It happens more often within long simulations (4 days+)
```

Comment:
I guess so, because maybe it was happening only with resource based elongation. I did not connected the two things but indeed the models I evaluate are all NOT res_...
I will check it.